.github/workflows/run_ubuntu_system_tests.yml

```diff
@@ -26,5 +26,5 @@ jobs:
         tar -xvzf ni-grpc-device-tests-linux-glibc2_31-x64.tar.gz
 
     - name: Run System Tests
-      run: ./SystemTestsRunner --gtest_filter=-NiRFSA*:NiRFSG*:NiDCPower*CalSelfCalibrate*
+      run: ./SystemTestsRunner --gtest_filter=-NiRFSA*:NiRFSG*:NiDCPower*CalSelfCalibrate*:NiXnetLINDriverApiTests.ConvertFramesToFromSignalsFromExample_CompareFrames_ValuesMatch
```

Comment:
I'm happy with this solution. But why is this used instead of a conditional GTEST_SKIP ([example](https://github.com/ni/grpc-device/blob/a47a73ebfd58fa1f621a1c7207586efc359a92e9/source/tests/system/nidigital_driver_api_tests.cpp#L51))?